Gemfile.lock

```diff
@@ -1,13 +1,13 @@
 GIT
-  remote: https://github.com/thepracticaldev/acts_as_follower.git
+  remote: git://github.com/thepracticaldev/acts_as_follower.git
```

Comment:
@maestromac I think you've left out the `Gemfile` with the modified link, because if I run `bundle install` after pulling this, `Gemfile.lock` goes back to `https` here

Reply:
Actually I can't seem to avoid it. That appears every time I bundle install.

Reply:
My machine for some reason had `bundle config github.https false`. Fixed with https://github.com/thepracticaldev/dev.to/commit/9dd4ec6bafbd33970df06f10fdcbf5e54ec0fd65

Reply:
thanks!